apps/i18n/signup/en_us.json

```diff
@@ -75,6 +75,7 @@
   "get_informational_emails": "Get informational emails about Code.org updates and opportunities (1-2/month)",
   "after_creating_your_account": "After creating your account, you will receive a set of onboarding emails to help you get started.",
   "go_to_my_account": "Go to my account",
+  "grades_taught": "What grade(s) do you teach?*",
```

Comment:
FYI - I followed Hannah's pattern for strings here: https://github.com/code-dot-org/code-dot-org/pull/58661